### DISCUSSION TITLE
Need help installing the toolkit solution

### DISCUSSION THREAD

**Discussion by YinkaOsho** [microsoft/finops-toolkit#812] [General]
I am new to Github and i have been tasked to install this solution and explore the Power BI Cost Management reports. I have downloaded the folder to my local machine and also opened the individual pbix. files however i think i am missing an installation of the solution which will allow all the pbix files available as one solution. please can you guide me through the process

**Comment by lundejd** [comment]
Did you run the FinOps hub deployment and use the documentation? This will walk you through the deployment step by step. https://microsoft.github.io/finops-toolkit/hubs  Just log into Azure with the right access and have a resource group ready for the deployment.

**Comment by arthurclares** [comment]
@YinkaOsho Do you still need help with this?

**Comment by AErmie** [comment]
❓ Is there a supported method for deploying the Hub/Toolkit using a CI/CD pipeline? The documentation seems to point to using the ARM template (via the Portal), and/or PowerShell. 

It would be nice to have a full CI/CD process, where we can deploy and upgrade the Hub as new versions are released. Any guidance on this approach would be appreciated.

As a bonus, it would also be welcomed if there was a Terraform version of the deployment as well.

**Reply by flanakin** [reply]
We share bicep and the JSON template. You can use either in a CI/CD pipeline.

There's a feature request for TF. I believe it's the top voted one right now. We started it. We just need to get it updated to the latest changes. (/cc @didayal-msft)

Sorry for the delay. This was hidden in the other question so we missed it.
